data/phoible.csv

```diff
@@ -89209,7 +89209,6 @@
 2478,"occi1239","oci","Occitan","Occitan (Gascon)","0069+032F+0065+0069+032F","i̯ei̯",NA,FALSE,"vowel","ea","0","-","-,+,-","-","-","-","+","+","0","+","-","-","-","-","-","0","0","-","0","0","0","+","+,-,+","-","+","-","+","-","-","+","-","-","-","0","-","-","0"
 2478,"occi1239","oci","Occitan","Occitan (Gascon)","0069+032F+0075","i̯u",NA,FALSE,"vowel","ea","0","-","-,+","-","-","-","+","+","0","+","-","-","-","-","-,+","+","-","-","0","0","0","+","+","-","+,-","-,+","+","-","-","+","-","-","-","0","-","-","0"
 2478,"occi1239","oci","Occitan","Occitan (Gascon)","0069+032F+0254","i̯ɔ",NA,FALSE,"vowel","ea","0","-","-,+","-","-","-","+","+","0","+","-","-","-","-","-,+","+","-","-","0","0","0","+","+,-","-","+,-","-,+","+,-","-","-","+","-","-","-","0","-","-","0"
-2478,"occi1239","oci","Occitan","Occitan (Gascon)","0069+032F+0254","i̯ɔ",NA,FALSE,"vowel","ea","0","-","-,+","-","-","-","+","+","0","+","-","-","-","-","-,+","+","-","-","0","0","0","+","+,-","-","+,-","-,+","+,-","-","-","+","-","-","-","0","-","-","0"
```

Comment:
Confused what this line deletion has to do with NA ISO codes... oh wait, is this a duplicate phoneme entry?  I wonder if it's worth checking the source to make sure there's not a phoneme missing... i.e., was the duplicate an accidental *addition* (accidentally duplicated a spreadsheet row) or an accidental *substitution* (intentionally duplicated a spreadsheet row, but failed to properly update the data in the copy)?  **EDIT** I looked at the inventory, it's almost certainly just a duplicated line (looking at the diphthongs w/ high-front onglides, there's no gap in the paradigm).